packages/interface-stream-muxer-compliance-tests/src/close-test.ts

```diff
@@ -67,6 +67,48 @@ export default (common: TestSetup<StreamMuxerFactory>) => {
       expect(dialer.streams).to.have.lengthOf(0)
     })
 
+    it('calling close closes streams', async () => {
+      let openedStreams = 0
+      const expectedStreams = 5
```

Comment:
Also, if a muxer is closed, calling `newStream` should probably throw?  That would need a test too.